cloudbuild.yaml

```
@@ -0,0 +1,25 @@
+  # Build the container image
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['build', '-t', 'gcr.io/${_PROJECT_ID}/${_IMAGE_NAME}:${_AUTO_TAG}', '-f', '${_DOCKERFILE}', '.']
+
+  # Push the container image to Container Registry
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['push', 'gcr.io/${_PROJECT_ID}/${_IMAGE_NAME}:${_AUTO_TAG}']
+
+  # Deploy to Cloud Run
+  - name: 'gcr.io/cloud-builders/gcloud'
+    args: ['run', 'deploy', '${_SERVICE_NAME}', '--image', 'gcr.io/${_PROJECT_ID}/${_IMAGE_NAME}:${_AUTO_TAG}', '--region', '${_REGION}', '--platform', 'managed', '--allow-unauthenticated']
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)